### PR TITLE
chore(module): add build optimisations module

### PR DESCRIPTION
**Pull request by loonpwn**
At a somewhat stable release so thought it would be good to add it here now: https://github.com/loonpwn/nuxt-build-optimisations

Let me know if any changes are required.

### CODE REVIEW

**Comment by loonpwn** [comment]
Hey @Geminii 

Thanks for the feedback. I've changed the category and added the icon.

Here's what it looks like if you're interested

image

**Comment by Geminii** [comment]
> Hey @Geminii
> 
> Thanks for the feedback. I've changed the category and added an icon.
> 
> Here's what it looks like if it helps
> 
> image

Looks perfect 👍

File: modules/build-optimisations.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: build-optimisations
+description: Nuxt.js v2 module to automatically make your builds faster.
+repo: loonpwn/nuxt-build-optimisations
+npm: nuxt-build-optimisations
+icon: build-optimisations.svg
+github: https://github.com/loonpwn/nuxt-build-optimisations
+website: https://github.com/loonpwn/nuxt-build-optimisations
+learn_more: ''
+category: 'Devtools'
```
Comment (Geminii) [inline modules/build-optimisations.yml:9]: Quote isn't needed around category :)

**Comment by loonpwn** [comment]
Hey @Geminii 

Thanks for your eye on that that, updated.

Keeping a uniform yaml file for all modules is a good idea. However, at the moment, it's a bit tedious for me to have to wait for a manual review to find that and I'm sure you have better things to do with your time.

With that in mind, I'm created a [PR](https://github.com/nuxt/modules/pull/108) to implement yamlllint as part of the CI process.

Also if you're asking people to add an icon, it would be good to document that in the readme.

**Comment by loonpwn** [comment]
Hey @pi0 

Was there an issue with this or was this accidentally closed? The other PR was irrelevant to this one

**Comment by pi0** [comment]
I'm so sorry @loonpwn indeed it was supposed to merged instead 🤦 Added in https://github.com/nuxt/modules/commit/feb1404c5c70fe0182ae52f977a0c887335d1937

**Pull request closed without merging**